src/common/support.js

```diff
@@ -255,7 +255,15 @@ function getSupportInfo(version, opts) {
 
   const languages = plugins
     .reduce((all, plugin) => all.concat(plugin.languages), [])
-    .filter(language => language.since && semver.gte(version, language.since))
+    .filter(language => {
+      if (language.since === null) {
+        return false;
+      }
+      if (language.since) {
+        return semver.gte(version, language.since);
+      }
+      return true;
+    })
```

Comment:
How about

```js
language.since
  ? semver.gte(version, language.since)
  : language.since !== null
```

?

Reply:
Yeah I originally had a ternary but it looked messy/hard to reason about so I switched it to if/else.